### ISSUE TITLE
Cannot reproduce baseline on DTU

### ISSUE THREAD

**Issue by PruneTruong** [dunbar12138/DSNeRF#54]
Hi, thanks for your nice work! I want to compare to it, and therefore I am trying to reproduce the results stated in the paper for NerF (baseline) on DTU, when training on 3, 6 and 9 views. According to the paper, you follow the evaluation protocol of PixelNerf. 
Therefore, I added in the load_dtu part: 

train_idx = [25, 22, 28, 40, 44, 48, 0, 8, 13]
exclude_idx = [3, 4, 5, 6, 7, 16, 17, 18, 19, 20, 21, 36, 37, 38, 39]
test_idx = [i for i in np.arange(49) if i not in train_idx + exclude_idx]
split_indices = {'test': test_idx, 'train': train_idx}

if args.train_sub is not None or args.train_sub > 0:
    train_idx = train_idx[:args.train_sub]

According to pixelnerf, test split is on 25 views, and the 3 training views correspond to index  [25, 22, 28], 6 to [25, 22, 28, 40, 44, 48] and 9 to [25, 22, 28, 40, 44, 48, 0, 8, 13]. Is this what you also did? 

For the config, as baseline nerf trained on 9 views, i used the following: 

expname = standard_nerf
dataset_type = dtu
train_sub = 9
render_test = True
N_rand = 1024
N_samples = 64
N_importance = 128
use_viewdirs = True
raw_noise_std = 1e0
no_ndc = True
colmap_depth = False
depth_loss = False
sigma_loss = False
depth_lambda = 0.
sigma_lambda = 0.
i_testset = 5000
i_video = 0
N_iters = 150000

Yet, when I train on dtu 9 views, I obtained bad results, far from the ones in the paper. For example, on scan40, psnr on the test set is 12. On Scan41, it is 11 and on scan45, 16. 

Am i missing something? Thanks for your help.

**Comment by dunbar12138** [comment]
Hi, thanks for your interest!

Yes, we follow the evaluation protocol of pixelNeRF and your train/test set up looks good to me. 

Did you use the given camera poses in the dataset? Inaccurate poses could lead to bad results.

**Comment by PruneTruong** [comment]
Could you provide the configs for baseline and DS-Nerf on dtu 3/9 views please? It would be much more convenient. Thanks a lot.

**Comment by dachao-ai** [comment]
> Could you provide the configs for baseline and DS-Nerf on dtu 3/9 views please? It would be much more convenient. Thanks a lot.

Hello, may I ask if your issue with DTU has been resolved? Could you please share the DTU dataset you created? Thank you very much.
